plugin/styleguide-plugin.js

```diff
@@ -79,8 +80,10 @@ StyleguidePlugin.prototype.apply = function apply(compiler) {
     </html>
     `;
 
-    // And emit that HTML template as 'styleguide.html'
-    compilation.assets['styleguide/index.html'] = {
+    const styleguidePath = this.options.basePath || 'styleguide';
```

Comment:
why do we specify a base path instead of the template name?
wouldn't it give the user even more control if he had a option called `dest` or `output` or `styleguide` which might be anything from 'styleguide.html` over webpack hashes like styleguide-[hash].html` to `my/funky/path/index.html`